lib/vmdb/loggers.rb

```diff
@@ -56,18 +45,33 @@ def self.create_logger(log_file_name, logger_class = VMDBLogger)
       configure_external_loggers
     end
 
-    private_class_method def self.create_broadcast_logger
-      create_container_logger || create_journald_logger
+    private_class_method def self.create_file_logger(log_file_name, logger_class)
+      log_file_name = ManageIQ.root.join("log", log_file_name)
+      progname      = progname_from_file(log_file_name)
+
+      logger_class.new(log_file_name, progname: progname)
```

Comment:
lol. Do we not do hashrockets for these?

Reply:
Oh yeah, I can change that - I tend to prefer this format for keyword params (which is what it was designed for), but not for Hashes.  Crystal does it this way, and I got mixed up haha.